docs/api/datasets.rst

```diff
@@ -113,6 +113,7 @@ Landsat
 .. autoclass:: Landsat3
 .. autoclass:: Landsat2
 .. autoclass:: Landsat1
+.. autoclass:: L8Biome
```

Comment:
Can you make a separate L8 Biome section for this dataset? Although it contains Landsat images, it's very different from these uncurated raw data loaders.